eslint/babel-eslint-parser/README.md

```diff
@@ -91,6 +91,21 @@ module.exports = {
 };
 ```
 
+**Monorepo configuration**
+
+This configuration is useful for monorepo as it avoids to repeat the configuration of Babel and ESLint on every package.
```

Comment:
```suggestion
If you are running ESLint in specific packages and not from the monorepo root, this configuration lets `@babel/eslint-parser` load the Babel configuration defined in the monorepo root.
```

(You don't need this option if the CWD is the monorepo root when running ESLint)

Reply:
I see. Maybe :
This configuration is useful for monorepo, when you are running ESLint on every package and not from the monorepo root folder, as it avoids to repeat the configuration of Babel and ESLint on every package.